dict_k_parameters_v_index.txt
"CAPPIS1980" => 207
"AWBPD" => 280
"BEM" => 102
"IPT" => 219
"DRI" => 167
"TENHW" => 185
"EROCEPA2022" => 80
"MWBGW" => 270
"ERDCH4KC2022" => 9
"GRCR" => 125
"CAPEXFED" => 97
"AI1980" => 34
"OWETFP" => 258
"WSOECLR" => 186
"INSR" => 121
"UNSR" => 124
"initWSO" => 187
"HRMI" => 33
"TAHW" => 192
"SCROTA" => 251
"FFLREOGRRM" => 148
"TEST" => 289
"EDELCM" => 217
"STRERDF" => 287
"ECRA22" => 142
"FIC" => 198
"IEAWBIF" => 268
"TAB" => 68
"DNCM" => 235
"FP" => 228
"DIC" => 173
"WSOELPR" => 194
"CAPEXRE1980" => 95
"TPM3I" => 35
"INVEODDI" => 175
"AUR" => 196
"CH4A1980" => 11
"GCH4PP" => 13
"LEEPA" => 249
"GPP" => 57
"PD" => 43
"GCWR" => 140
"FNE1980" => 84
"OWELE" => 250
"STR" => 71
"PFTJ80" => 180
"LER80" => 155
"TCO2ETN2O" => 22
"TELLM" => 190
"EGTRF2022" => 49
"TYLD" => 184
"NSR" => 117
"CBECLE" => 139
"initLE" => 245
"CTF" => 136
"TPS" => 275
"WVC1980" => 36
"WPP" => 74
"DNC80" => 232
"EDEFRA" => 215
"FUELER" => 144
"OGRR80" => 158
"PAEAWBF" => 290
"DAT" => 169
"LEA" => 243
"SWI1980" => 172
"GITRO" => 45
"FECCT" => 107
"MFAM" => 143
"IPRVPSS" => 260
"TI" => 267
"sFCUTLOFC" => 109
"OPEXFED" => 98
"initSTE" => 286
"initSOTR" => 288
"CO2ARA" => 141
"NCUT" => 110
"OBWA2022" => 14
"RCUT" => 90
"TFFLR" => 129
"UDT" => 161
"CTPIS" => 203
"IEFT" => 120
"INVEOSWI" => 168
"STEERDF" => 285
"GEIC" => 59
"GN2OPP" => 7
"SRAT" => 119
"UT" => 122
"XETAC2022" => 263
"DACCO22100" => 20
"SRI" => 176
"CO2CEACY" => 133
"CNED" => 75
"TPTH2" => 101
"GDPPEROCCLRM" => 182
"KWEPKGH2" => 100
"TAWC" => 70
"KCH4KC1980" => 8
"LCPIS1980" => 208
"SAT" => 177
"MAT" => 4
"CMFR" => 230
"FUATA" => 259
"ALIS" => 25
"GSF2022" => 56
"ITRO1980" => 44
"MTPEJCE" => 105
"FCI" => 202
"MWDB" => 63
"GEPA" => 237
"ICPT" => 174
"ISCEGA1980" => 28
"SOWLCO2" => 15
"PPU" => 164
"EETF2022" => 48
"FSRT" => 118
"TCE" => 111
"CO2C2022" => 134
"RDCH4KC" => 10
"INELOK" => 199
"TOED" => 226
"SSP2FA2022F" => 240
"MNFCO2PP" => 76
"ROCFSP" => 153
"ITRO2022" => 46
"NIEE" => 81
"FGDC2022" => 47
"SSP2LMA" => 131
"THPA" => 291
"ED1980" => 216
"NBOM" => 115
"WA1980" => 31
"BITRW" => 61
"FF80" => 146
"TCO2PTCH4" => 17
"MFM" => 241
"EDROTA2022" => 254
"FETACPET" => 52
"DROTA1980" => 253
"WDP" => 73
"KCKRM" => 145
"GDPP1980" => 66
"EROCFSP" => 152
"KN2OKF1980" => 2
"initEGDPP" => 233
"ALGAV" => 26
"PREAWBF" => 273
"IPTCB" => 126
"EUEPRUNEFF" => 86
"GDDP" => 55
"ROCTCO2PT" => 79
"SVDR" => 32
"ROCECLR80" => 197
"GDPTL" => 256
"initFOLA" => 147
"KAPPA" => 225
"SPS" => 281
"PH" => 166
"FETPO" => 67
"initWF" => 189
"XETAC2100" => 264
"EGB22" => 156
"ERDN2OKF2022" => 3
"initPA" => 248
"INVEOIN" => 171
"N2OA1980" => 5
"NK" => 200
"DNCA" => 231
"IIEEROTA" => 262
"ETGBW" => 53
"initEPA" => 238
"GCF" => 58
"GCO2PP" => 21
"NBBM" => 116
"GDPPEFRACA" => 221
"ULP" => 154
"CO2A1850" => 18
"OW2022" => 163
"GREF" => 88
"TW" => 269
"SWC1980" => 93
"LAMBDA" => 223
"LECO2A1980" => 16
"initNHW" => 181
"SC1980" => 252
"GDPOSR" => 65
"CO2RHFC" => 135
"FW" => 229
"RDN2OKF" => 1
"TRSS1980" => 29
"IT" => 123
"GFCO2SCCS" => 77
"MIROTA2022" => 257
"OPEXRED" => 96
"WCF" => 72
"LREC" => 92
"ROCFP" => 150
"FRA1980" => 218
"EIPF" => 278
"NRD" => 279
"GLSU" => 27
"WVWVF" => 39
"AFMCM" => 112
"PFTJ" => 179
"GFNE" => 82
"OWEACY" => 162
"TINT" => 51
"OSF1980" => 64
"AFGDP" => 137
"DDI1980" => 165
"NLFEC" => 108
"CAPPUS1980" => 209
"FT1980" => 54
"UPTCB" => 127
"LN2OA" => 6
"TCO2ETCO2" => 24
"CTA2022" => 255
"GFRA" => 151
"AI" => 276
"FCG" => 157
"REFF2022" => 89
"RWER" => 191
"PUELPR" => 195
"FRACAM" => 220
"DRPS" => 274
"EIP" => 277
"IPR1980" => 261
"CC1980" => 205
"TAS" => 178
"MRIWI" => 170
"FREH" => 99
"DRC" => 159
"GENLPR" => 183
"GEFR" => 236
"EKHPY" => 106
"PESTF" => 283
"CBCEFRA" => 214
"MGDB" => 62
"FADFS" => 247
"TWHPEJCE" => 104
"EMCUC" => 206
"FCO2SCCS2022" => 78
"MRS1980" => 30
"CTPUS" => 204
"TEGR" => 201
"OO1980" => 222
"USPIS2022" => 210
"GWEAWBGWF" => 271
"GFNRM" => 149
"LEG" => 244
"OWECCM" => 212
"REFF1980" => 87
"CYRA" => 128
"TRSA1980" => 42
"MLEM" => 246
"TC" => 114
"TAOC" => 69
"CRDRA" => 132
"WSOEFRA" => 227
"OWWV" => 37
"FUESQ" => 160
"WVF1980" => 38
"WFEH" => 40
"OWELCM" => 213
"INEQ1980" => 60
"TCO2ETCH4" => 23
"LEMAX" => 242
"PRUN" => 188
"LCH4A" => 12
"AP" => 284
"CRDSWC" => 94
"TCRD" => 282
"EPTF2022" => 50
"CCCSt" => 19
"TCTB" => 138
"NLPR80" => 193
"USPUS2022" => 211
"TAHI" => 239
"TDI" => 266
"LAUS1980" => 224
"DRDI" => 265
"ECRUNEFF" => 85
"EH1980" => 41
"EFPP" => 103
"SFU" => 130
"FNE2022" => 83
"TCFFFNEU" => 113
"DNCG" => 234
"RECT" => 91
"TPR" => 272


parameters_default_values.txt
p = [0.01,0.11,0.0,5.0,1.052,95.0,5.0,0.05,0.0,0.01,
2.5,7.5,5.0,1.35,1.0,60.0,2.75,2200.0,95.0,0.0,
7.9,7.0,23.0,1.0,0.7,0.3,510.0,12.0,0.01,0.0015,
0.4,4.0,333.0,55.0,0.95,2.0,0.18,0.9,3.0,0.0006,
0.0,0.01,5.0,0.4,0.3,0.3,0.0,0.0,0.0,0.0,
5.0,0.5,0.0,0.3,10.0,0.0,200.0,0.75,0.0,0.61,
0.2,1.0,1.0,0.9,-0.06,6.4,0.5,1.0,1.0,1.0,
0.03,0.9,10.0,20.0,0.033,0.5,0.2,0.0,-0.003,0.002,
0.01,0.5,0.03,0.0,10.0,3.0,0.065,0.5,0.23,3.0,
3.0,40.0,10.0,0.2,7.0,0.001,0.7,0.02,0.0,40.0,
10.0,0.0,0.345,278.0,24.0,8.0,3.0,40.0,0.5,8.0,
0.03,1.35,240.0,0.02,0.015,0.005,0.02,1.0,1.0,10.0,
0.7,0.05,0.02,-1.5,0.0,1.0,1.0,5.0,0.2,20.0,
1.0,0.05,0.3,420.0,65.0,500.0,0.05,0.0,-0.03,0.05,
1.0,400.0,2.0,0.02,24.0,88450.0,1100.0,-5.0,0.1,0.01,
0.1,0.0,0.002,0.05,0.004,5.0,0.1,0.004,0.05,-0.001,
10.0,-0.3,1.35,1.0,1.0,0.0,1.0,-0.6,1.2,1.07,
-0.26,1.0,0.4,0.25,-0.6,1.0,1.0,0.24,1.0,1.0,
2.0,-0.1,0.0,2.3,-0.03,1.05,0.5,1.0,1530.0,5.0,
0.015,5.0,0.85,0.2,0.05,0.05,0.02,1.0,-0.5,0.3,
4.0,0.0,1.5,1.5,1.0,1.7,59250.0,15.0,5350.0,0.0,
0.0,0.2,-0.1,-0.8,5.0,1.0,0.5,0.9,1.0,0.65,
-0.2,28087.0,0.7,3060.0,0.3,1.0,-2.5,20.0,0.5,0.01,
0.0,4.3,6.4,0.14,1.2,0.0,0.0,0.0,10.0,1.0,
1.6,85.0,0.001,0.15,67.0,1.1,0.8,62.0,0.75,-0.02,
0.5,0.3,0.01,0.003,9145.0,15.0,0.0,-0.1,0.3,1.0,
1.2,-0.1,0.0,0.0,0.5,15.0,0.5,-0.6,1.0,0.2,
-0.58,0.02,6.0,0.7,3.0,0.6,30.0,0.0,30.0,9.0,
0.3,10.0,-15.0,0.02,1.0,1.3,-1.0,0.6,10.0,0.5,
0.8,]

EASensitivityTest/parameters.jl
_params = Dict{Symbol,Float64}(
    # climate
    :initFOLA => 1100,
    :initNHW => 2,
    :initWSO =>0.5,
    :initWF => 1530,
    :initEGDPP => 6.4,
    :initEPA => 0,
    :initLE => 67,
    :initPA => 62,
    :initSTE => 1.3,
    :initSOTR => 0.6, 
    :ERDN2OKF2022 => 0, # 0.01 in GL,
    :RDN2OKF => 0.01,
    :KN2OKF1980 => 0.11,
    :N2OA1980 => 1.052,
    :MAT => 5,
    :LN2OA => 95,
    :GN2OPP => 5,
    :ERDCH4KC2022 => 0, # 0.01 in GL,
    :RDCH4KC => 0.01,
    :KCH4KC1980 => 0.05,
    :CH4A1980 => 2.5,
    :LCH4A => 7.5,
    :GCH4PP => 5,
    :OBWA2022 => 1.35,
    :SOWLCO2 => 1,
    :LECO2A1980 => 60,
    :CO2A1850 => 2200,
    :TCO2PTCH4 => 2.75,
    :DACCO22100 => 0, # 8 in GL,
    :GCO2PP => 7.9,
    :TCO2ETN2O => 7,
    :TCO2ETCH4 => 23,
    :TCO2ETCO2 => 1,
    :ALGAV => 0.3,
    :ALIS => 0.7,
    :GLSU => 510,
    :ISCEGA1980 => 12,
    :TRSS1980 => 0.01,
    :MRS1980 => 0.0015,
    :WA1980 => 0.4,
    :SVDR => 4,
    :AI1980 => 55,
    :TPM3I => 0.95,
    :HRMI => 333,
    :OWWV => 0.18,
    :WVC1980 => 2,
    :WVF1980 => 0.9,
    :WVWVF => 3,
    :WFEH => 0.0006,
    :EH1980 => 0,
    :PD => 5,
    :TRSA1980 => 0.01,
    :CCCSt => 95,
    # demand
    :BITRW => 0.2,
    :EETF2022 => 0, # 0.02 in GL,
    :EGTRF2022 => 0, # 0.01 in GL,
    :EPTF2022 => 0, # 0.02 in GL,
    :ETGBW => 0, # 0.2 in GL,
    :FETACPET => 0.5,
    :FETPO => 0.5, # 0.8 in GL,
    :FGDC2022 => 0, # 0.1 in GL,
    :FT1980 => 0.3,
    :GCF => 0.75,
    :GDPOSR => -0.06,
    :GDPP1980 => 6.4,
    :GDDP => 10,
    :GEIC => 0, # 0.02 in GL,
    :GITRO => 0.3,
    :GPP => 200,
    :GSF2022 => 0,
    :INEQ1980 => 0.61,
    :ITRO1980 => 0.4,
    :ITRO2022 => 0.3,
    :OSF1980 => 0.9,
    :MATGF => 0.64,
    :MATWF => 0.39,
    :MWDB => 1,
    :MGDB => 1,
    :STR => 0.03,
    :TAB => 1,
    :TAOC => 1,
    :TAWC => 1,
    :TINT => 5,
    :WCF => 0.9,
    :WDP => 10,
    :WPP => 20,
    # energy
    :MNFCO2PP => 0.5,
    :FCO2SCCS2022 => 0,
    :GFCO2SCCS => 0.2, # 0.9 in GL,
    :CCCSt => 95,
    :ROCTCO2PT => -0.003,
    :EROCEPA2022 => 0.002, # 0.004 in GL,
    :NIEE => 0.01,
    :GFNE => 0.5, # 1 in GL,
    :FNE2022 => 0.03,
    :FNE1980 => 0,
    :EUEPRUNEFF => 3,
    :ECRUNEFF => 10,
    :GREF => 0.5, # 1 in GL,
    :REFF2022 => 0.23,
    :REFF1980 => 0.065,
    :RCUT => 3,
    :RECT => 3,
    :LREC => 40,
    :SWC1980 => 10,
    :CRDSWC => 0.2,
    :CAPEXRE1980 => 7,
    :CAPEXFED => 0.7,
    :OPEXRED => 0.001,
    :OPEXFED => 0.02,
    :CNED => 0.033,
    :FREH => 0,
    :KWEPKGH2 => 40,
    :TPTH2 => 10,
    :BEM => 0,
    :EFPP => 0.345,
    :TWHPEJCE => 278,
    :MTPEJCE => 24,
    :EKHPY => 8,
    :FECCT => 3,
    :NLFEC => 40,
    :sFCUTLOFC => 0.5,
    :NCUT => 8,
    :TCE => 0.03,
    :AFMCM => 1.35,
    :TCFFFNEU => 240,
    :TC => 0.02,
    # finance
    :FSRT => 1,
    :GRCR => 0,
    :IEFT => 10,
    :INSR => 0.7,
    :IPTCB => 1,
    :IT => 0.02,
    :NBBM => 0.005,
    :NBOM => 0.015,
    :NSR => 0.02,
    :SRAT => 1,
    :UNSR => -1.5,
    :UPTCB => 1,
    :UT => 0.05,
    # foodland
    :AFGDP => 0.05,
    :CBECLE => -0.03,
    :CO2ARA => 1,
    :CO2C2022 => 420,
    :CO2CEACY => 0.3,
    :CO2RHFC => 65,
    :CRDRA => 0.05,
    :CTF => 500,
    :CYRA => 5,
    :DRC => 0.05,
    :ECRA22 => 400,
    :EGB22 => 5,
    :EROCFSP => 0,
    :FCG => 0.1,
    :FFLREOGRRM => -5,
    :FU80 => 61,
    :FUELER => 0.02,
    :FUESQ => -0.001,
    :GCWR => 0.05, # 0.2 in GL,
    :GFNRM => 0.1, # 0.5 in GL,
    :GFRA => 0.1, # 0.5 in GL,
    :KCKRM => 24,
    :LER80 => 0.004,
    :MFAM => 2,
    :OGRR80 => 0.004,
    :OW2022 => 1.35,
    :OWEACY => -0.3,
    :ROCFP => 0.01,
    :ROCFSP => 0.002,
    :SFU => 20,
    :SSP2LMA => 1,
    :TCTB => 0,
    :TFFLR => 0.2,
    :UDT => 10,
    :ULP => 0.05,
    # inventory
    :DAT => 1.2,
    :DDI1980 => 1,
    :DIC => 0.4,
    :DRI => 1,
    :ICPT => 0.25,
    :MRIWI => 1.07,
    :OO => 28087,
    :PH => 0,
    :PPU => 1,
    :SAT => 1,
    :INVEODDI => -0.6,
    :INVEOIN => -0.26,
    :INVEOSWI => -0.6,
    :SRI => 1,
    :SWI1980 => 1,
    :TAS => 0.24,
    # labourmarket
    :AUR => 0.05,
    :FIC => 1,
    :GDPP1980 => 6.4, # Should be the same in Population
    :GDPPEROCCLRM => -0.1,
    :GENLPR => 0,
    :NLPR80 => 0.85,
    :PFTJ => 1,
    :PFTJ80 => 1,
    :PRUN => 1, # Taken from Vensim table
    :PUELPR => 0.05,
    :ROCECLR80 => 0.02,
    :RWER => 0.015,
    :TAHW => 5,
    :TENHW => -0.03,
    :TELLM => 5,
    :TYLD => 2.3,
    :WSOECLR => 1.05,
    :WSOELPR => 0.2,
    # other
    :TEGR => 4,
    :INELOK => -0.5,
    :NK => 0.3,
    # output
    :CAPPIS1980 => 59250,
    :CAPPUS1980 => 5350,
    :CC1980 => 1,
    :CTPIS => 1.5,
    :CTPUS => 1.5,
    :EMCUC => 1.7,
    :FCI => 0,
    :LAUS1980 => 3060, # Taken from Vensim table
    :LCPIS1980 => 15,
    :OW2022 => 1.35, # Taken from Climate sector
    :OWECCM => 0.2,
    :OWELCM => -0.1,
    :USPIS2022 => 0, # 0.01 in GL,
    :USPUS2022 => 0, # 0.01 in GL,
    :CBCEFRA => -0.8,
    :CU1980 => 0.8,
    :ED1980 => 1,
    :EDEFRA => 5,
    :EDELCM => 0.5,
    :FRA1980 => 0.9,
    :FRACAM => 0.65,
    :GDPP1980 => 6.4, # It should be the same as in the Labour and market sector
    :GDPPEFRACA => -0.2,
    :IPT => 1,
    :JOBS1980 => 1600,
    :KAPPA => 0.3,
    :LAMBDA => 0.7, # Calculated as 1-KAPPA
    :MA1980 => 0.25,
    :OG1980 => 0.06,
    :OO1980 => 28087, # Taken from Vensim table
    :PCORPIS => 2.3,
    :PCORPUS => 2.3,
    :TOED => 1,
    :WSOEFRA => -2.5,
    # population
    :CMFR => 0.01,
    :DNC80 => 4.3,
    :DNCA => 0,
    :DNCG => 0.14,
    :DNCM => 1.2,
    :EIP => 30,
    :EIPF => 0,
    :FP => 20,
    :FW => 0.5,
    :FADFS => 0.8,
    :GEFR => 0, # 0.2 in GL,
    :GEPA => 0,
    :LEA => 0.001,
    :LEEPA => 0.75,
    :LEG => 0.15,
    :LEMAX => 85,
    :MFM => 1.6,
    :MLEM => 1.1,
    :ORDER => 10,
    :OW2022 => 1.35,
    :OWELE => -0.02,
    :SSP2FA2022F => 1,
    :TAHI => 10,
    # public
    :CTA2022 => 9145,
    :CTPIS => 1.5, # Taken from Output sector
    :EDROTA2022 => 0.003,
    :DROTA1980 => 0.01,
    :FUATA => 0.3,
    :GDPTL => 15,
    :IIEEROTA => -0.1,
    :IPR1980 => 1.2,
    :IPRVPSS => 1,
    :IPT => 1, # Taken from Output sector
    :MIROTA2022 => 0, # 0.005 in GL,
    :OWETFP => -0.1,
    :SC1980 => 0.3,
    :SCROTA => 0.5,
    :XETAC2022 => 0,
    :XETAC2100 => 0,
    :OW2022 => 1.35,
    # wellbeing
    :AI => 0.6,
    :AP => 0.02,
    :AWBPD => 9,
    :DRDI => 0.5,
    :DRPS => 0.7,
    :EIP => 30,
    :EIPF => 0,
    :GWEAWBGWF => -0.58,
    :IEAWBIF => -0.6,
    :MWBGW => 0.2,
    :NRD => 30,
    :PESTF => -15,
    :PAEAWBF => 0.5,
    :PREAWBF => 6,
    :SPS => 0.3,
    :STEERDF => 1,
    :STRERDF => -1,
    :TCRD => 10,
    :TDI => 15,
    :TEST => 10,
    :TI => 0.5,
    :THPA => 0.8,
    :TPR => 0.02,
    :TPS => 3,
    :TW => 1,
   
)

getparameters() = copy(_params)
